src/content/docs/en/recipes/reading-time.mdx

```diff
@@ -85,4 +85,32 @@ Create a [remark plugin](https://github.com/remarkjs/remark) which adds a readin
       <slot />
     </body>
   </html>
-  ```
\ No newline at end of file
+  ```
+  
+  For [Content Collections](/en/guides/content-collections/), you can access the `remarkPluginFrontmatter` from the `render()` function of a collection entry.
```

Comment:
```suggestion
 If your blog posts are stored in a [content collection](/en/guides/content-collections/), access the `remarkPluginFrontmatter` from the `entry.render()` function.
```
Don't use "you can" if it really means "you must"!  :smile:

And, I think we also need to  make it clear that these are two different options, and there's no "if"/conditional until you get to this second option.

I think I'd propose:

> 4. If your blog posts are stored in a content collection.... (then all this new stuff here)
>
> To add the remark plugin to pages outside of a collection... (then the old stuff)

What do you think?